sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

```diff
@@ -0,0 +1,343 @@
+    "\n",
+    "print('SavedModel exported to {}'.format(second_model_path))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we need to create a model archive file containing the exported model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a model archive file\n",
+    "\n",
+    "SageMaker models need to be packaged in `.tar.gz` files. When your endpoint is provisioned, the files in the archive will be extracted and put in `/opt/ml/model/` on the endpoint. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tar -C \"$PWD\" -czf mobilenet.tar.gz mobilenet/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Upload the model archive file to S3\n",
+    "\n",
+    "We now have a suitable model archive ready in our notebook. We need to upload it to S3 before we can create a SageMaker Model that. We'll use the SageMaker Python SDK to handle the upload."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.session import Session\n",
+    "\n",
+    "model_data = Session().upload_data(path='mobilenet.tar.gz', key_prefix='model')\n",
+    "print('model uploaded to: {}'.format(model_data))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a SageMaker Model and Endpoint\n",
+    "\n",
+    "Now that the model archive is in S3, we can create a Model and deploy it to an \n",
+    "Endpoint with a few lines of python code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow.serving import Model\n",
+    "\n",
+    "# Use an env argument to set the name of the default model.\n",
+    "# This is optional, but recommended when you deploy multiple models\n",
+    "# so that requests that don't include a model name are sent to a \n",
+    "# predictable model.\n",
+    "env = {'SAGEMAKER_TFS_DEFAULT_MODEL_NAME': 'mobilenet_v2_140_224'}\n",
+    "\n",
+    "model = Model(model_data=model_data, role=sagemaker_role, framework_version=1.11, env=env)\n",
+    "predictor = model.deploy(initial_instance_count=1, instance_type='ml.c5.xlarge')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Make predictions using the endpoint\n",
+    "\n",
+    "The endpoint is now up and running, and ready to handle inference requests. The `deploy` call above returned a `predictor` object. The `predict` method of this object handles sending requests to the endpoint. It also automatically handles JSON serialization of our input arguments, and JSON deserialization of the prediction results.\n",
+    "\n",
+    "We'll use these sample images.\n",
```

Comment:
nit: I'd end this with a colon rather than a period

Reply:
ok